Added a page for integrating a project with fly postgres (#1)
pmbanugo/flycd-docs: src/content/docs/guides/integrating-with-fly-postgres.md

---

> title: Integrating a project with fly postgres

Reviewer: ```suggestion
title: Integrate A Project with Fly Postgres Database
```

Let's try this title. For the other ones, please use a similar letter casing (similar to what's on Notion)

> description: A guide about Integrating a project with flycd postgres.

Reviewer: ```suggestion
description: A guide about using Fly Postgres for preview & production databases.
```

> ---

Reviewer: I would like to have a different title in the sidebar. See this link for [reference](https://starlight.astro.build/reference/frontmatter/#sidebar).

```suggestion
sidebar:
  label: Use Fly Postgres DB
---
```

Each preview app will get a unique `DATABASE_URL` when they’re first created. Production apps get just the same `DATABASE_URL` for every update/deployment.

## How to connect a FlyCD project to a Fly Postgres instance

First ensure you have created the Fly Postgres cluster, then follow the steps below:

1. Go to your [FlyCD dashboard](https://flycd.dev/dashboard).
2. Click on the project you want to add integration for Fly Postgres. This should take you to the project’s info page.
3. Click **Settings** at the top navigation, and you’ll be taking to the project’s settings page.
4. Click **Fly Postgres** on the left side navigation, and you’ll be shown the **Fly Postgres** settings page.

![Project settings page!](../../../assets/images/project-settings-page.webp)

5. Now enter the name of the Fly Postgres app you want to attach to the project.
6. Then click the **Update** button to save your changes.

Once the changes are saved, new deployments for the project will use the new settings/configuration.

> > The production and preview Postgres app can be the same. Ideally, you’d want to have a development cluster and a separate production cluster. Use the development cluster for all your preview deployment.

Reviewer: ```suggestion
The production and preview Postgres app can be the same. Ideally, you’d want to have a development cluster and a separate production cluster. Use the development cluster for all your preview deployment.
```

I'd like to remove this block-quote for now
